### ISSUE TITLE
Design | Ch33 Direct Deposit | Update Figma and Use Case Documentation for our Single Form

### ISSUE THREAD

**Issue by mtcA6** [department-of-veterans-affairs/va.gov-team#91917]

**Comment by mtcA6** [comment]
@AshleyEBush @jfinnerty13 What are y'alls thoughts on updating the Direct Deposit figma in next sprint? 

The ch33 updates were pushed AND there's the Verify Your Enrollment Team work that also made it to production between september and december last year

**Comment by jfinnerty13** [comment]
@mtcA6 where does this fall in priority with Int Phone Numbers, Confirmation Emails, Profile Secondary Research, and ILER IES?

**Comment by mtcA6** [comment]
Well, it's none of those, we're pushing those two additional alerts on DD page, we can refactor this ticket to include: 

- the Ch33 changes, 
- the VYE changes,
- our dependent alert
- our 422 alert

Doesn't have to be URGENT it can be in 2-3 sprints time when we've pushed the 422 and dependent alert. I wanted this on your radar

**Comment by mtcA6** [comment]
Or we don't refactor and we handle this in a different way after we have that mid April 15 talk about process flow future proofing.

**Comment by jfinnerty13** [comment]
@AshleyEBush and I were talking about a good candidate to try out something new so let's save this one for the future proofing exercise if it's non urgent please :) @mtcA6

**Comment by mtcA6** [comment]
Ok perfect, I'll tentatively put this in sprint 8

**Comment by raquelou** [comment]
Hello! @oddballpete has been in conversation with your team(AuthEx) about our team (IIR) taking on the work outlined in this ticket. I have a few questions and recommendations for next steps to ensure that our team (IIR) would deliver assets that are aligned with your (AuthEx) expectations. 

1. From my quick review of the Figma designs, it seems like the current designs are organized in a few different ways in figma. One way is displaying the screens as part of a user flow. The second way is displaying screens with descriptions of scenarios. Does your team (AuthEx) have a preferred way they want the figma pages/workflows organized? Do they want both methods they currently have?
2.  I recommend that the activities in these tickets be addressed as follows:
    a. Design the user flow first
    b. Review user flow with AuthEx team to ensure all use cases are met
    c. Build the figma pages using design components based on user flows
3. When they say "Capture all use cases" they mean capture them in the figma workflow right?

**Comment by oddballpete** [comment]
cc: @jfinnerty13

**Comment by jfinnerty13** [comment]
Hi @raquelou ! I'd recommend that we have a 30 min kick-off to discuss. Unfortunately we have the old files, and the new files, and neither match at the moment and it'd be a lot easier to talk in person. Are you ok with us setting up time or would you like to? Thanks!

**Comment by raquelou** [comment]
Yes, this would be great. I am fine with you setting up a time. Please make sure to add @DavidPearlUX and @oddballpete
